examples/pipelines/utils/upload_data/submit.py

```diff
@@ -183,14 +204,16 @@ def fl_cross_silo_upload_data():
                 type=AssetTypes.URI_FOLDER,
                 mode="mount",
                 path=custom_fl_data_path(
-                    silo_config.datastore, f"{args.example.lower()}/raw_train_data"
+                    silo_config.datastore,
+                    f"{args.example.lower() + '_vertical' if args.vertical else ''}/raw_train_data",
```

Comment:
I think it could be clearer if we named this a distinct example entirely, so instead of having 2 arguments, one --vertical and one --example, you'd have --example CCFRAUD_VERTICAL